backend/services/authService.js

```diff
@@ -118,6 +129,18 @@ module.exports = {
 		return true;
 	},
 
+	readUserAuth: async function(loginUser, targetUser) {
+		const isHRM = await isHumanResourceManager(loginUser, targetUser).catch(err => {throw err});
```

Comment:
`isHumanResourceManager` would be more appropriate than just `isHRM`. This could cause confusion later on.